### PR TITLE
pam.conf: clarify default action for unspecified return codes

### PR DESCRIPTION
**Pull request by jsquyres**
Add short blurbs explaining that if a return code is not specified in
the "[value1=action1 value2=action2 ...]" form and "default=action" is
not specified, that return code's action defaults to "bad".

### CODE REVIEW

File: doc/man/pam.conf-syntax.xml
Comment (t8m) [inline doc/man/pam.conf-syntax.xml:271]: I think this is a little bit confusing given the context. It also repeats the information given below. So I'd just add `This is the default action for all return codes.` here.

**Review by t8m — CHANGES_REQUESTED** [review]

File: doc/man/pam.conf-syntax.xml
Comment (jsquyres) [reply doc/man/pam.conf-syntax.xml]: Done!

**Comment by jsquyres** [comment]
I made the requested changes; thanks!

Can someone approve the CI workflow on this PR?  I'm a first-time contributor, so Github won't automatically launch CI until a maintainer approves it.  Thank you!

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
Merged. Thank you for the contribution.

**Pull request merged**
